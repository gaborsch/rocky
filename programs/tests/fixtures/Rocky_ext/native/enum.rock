from java, lang, annotation play ElementType
from java time temporal play ChronoField

say "ElementType is " + ElementType
let M be METHOD on ElementType
say "METHOD is " + M

say "ChronoField is " + ChronoField
let C be MINUTE_OF_DAY on ChronoField
say "MINUTE_OF_DAY is " + C
say "MINUTE_OF_DAY.isTimeBased is " + isTimeBased on C
say toString on C