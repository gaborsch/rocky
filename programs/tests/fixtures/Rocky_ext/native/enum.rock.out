ElementType is java.lang.annotation.ElementType
METHOD is METHOD
ChronoField is java.time.temporal.ChronoField
MINUTE_OF_DAY is MinuteOfDay
MINUTE_OF_DAY.isTimeBased is true
MinuteOfDay
